cmd/build

```diff
@@ -63,114 +75,14 @@ if [ "$target_commit" != "$local_version" ]; then
     false
 fi
 
-if [ -n "$in_container" ]; then
-    # Run in container for pylint versioning.
-    echo Running style checks...
-    bin/check_style
-
-    echo 'DOCKER_OPTS="-s vfs -D -l debug"' > /etc/default/docker
-    service docker start
-    sleep 3
-    echo Importing current local version of daq...
-    ROOT=/root/daq
-    rm -rf $ROOT
-    mkdir -p $ROOT
-    cd /daq
-    for subdir in $copydirs; do
-        echo Copying subdir $subdir...
-        cp -a $subdir $ROOT/
-    done
-    cd $ROOT
-    if [ "$do_images" != load ]; then
-        rm -rf build
-    fi
-    echo Now running in `pwd`
-fi
-
-sudo rm -rf $build_root .build_hash .build_files
-mkdir -p $build_root
-
-build_list_file=$build_root/build_list.txt
-echo > $build_list_file
-(
-    shopt -s nullglob
-
-    function include {
-        echo Including tests from $1
-        target=$1
-        source $1
-    }
-
-    function build {
-        if [ -d $1 ]; then
-            echo Including build files from $1
-            echo $1/Dockerfile.* >> $build_list_file
-        else
-            echo Missing build directory $1
-            false
-        fi
-    }
-
-    function add {
-        true
-    }
-    function remove {
-        true
-    }
-
-    include ${host_tests:-misc/host_tests.conf}
-)
-docker_files=$(<$build_list_file)
+CHECK_EXISTING=$CHECK_EXISTING bin/docker_build $inline
```

Comment:
I would convert this to a command-line arg, unless there's a reason to be env arg.

Reply:
I just think this is a lot cleaner and easier to use an env var right here.